### DISCUSSION TITLE
Cycling through overlapping hints

### DISCUSSION THREAD

**Discussion by andrew222651** [brookhong/Surfingkeys#1873] [Q&A] [answered]
Is there a keyboard shortcut for changing the z-index of overlapping hints so that ones that are covered go on top? VimFx has this: https://github.com/akhodakivskiy/VimFx/blob/master/documentation/commands.md#:~:text=Press%20%3Cc%2Dspace%3E%20and%20%3Cs%2Dspace%3E%20to%20switch%20which%20one%20should%20be%20on%20top.

**Comment by Madd0g** [comment, marked as answer]
shift key does that
